protobuf_definitions/message_formats.proto

```diff
@@ -363,6 +363,44 @@ message PositionEstimate {
   repeated NavigationSensorStatus navigation_sensors = 11; // List of available sensors with status
 }
 
+/*
+ * Waypoint used to describe a location and depth on a map.
+ */
+message Waypoint {
+  uint32 id = 1; // waypoint id
+  double latitude = 2; // latitude (decimal degrees)
```

Comment:
Why not reuse the `LatLongPosition` position type for these? Will also make it more convenient as the same type (and supporting functions/mapping/validation) can operate on the same `LatLongPosition` type.